napari_arboretum/plugin.py

```diff
@@ -24,22 +25,28 @@ class Arboretum(QWidget, TrackPropertyMixin):
     def __init__(self, viewer: napari.Viewer, parent=None):
         super().__init__(parent=parent)
         self.viewer = viewer
+        self.title = QLabel()
         self.plotter: TreePlotterQWidgetBase = VisPyPlotter()
         self.property_plotter: PropertyPlotterBase = MPLPropertyPlotter(viewer)
 
         # Set plugin layout
         layout = QGridLayout()
-        # Make the tree plot a bigger than the property plot
-        for row, stretch in zip([0, 1], [2, 1]):
-            layout.setRowStretch(row, stretch)
-        self.setLayout(layout)
 
-        # Add tree plotter
         row, col = 0, 0
+        # Add title
+        layout.addWidget(self.title, row, col, Qt.AlignHCenter)
+        self.title.setText("Arboretum")
+
+        # Add tree plotter
+        row = 1
         layout.addWidget(self.plotter.get_qwidget(), row, col)
         # Add property plotter
-        row = 1
+        row = 2
         layout.addWidget(self.property_plotter.get_qwidget(), row, col)
```

Comment:
Now that we've dropped support for `python=3.7` we could take advantage of the 🦭 operator and do 
```suggestion
        layout.addWidget(self.property_plotter.get_qwidget(), row:=2, col)
```